Gemfile

```diff
@@ -84,6 +83,7 @@ group :test do
   gem 'webmock', '~> 3.5'
   gem 'faker', '~> 1.9'
   gem 'aws-sdk-s3', '~> 1.21'
+  gem 'azure-storage', '~> 0.15.0.preview'
```

Comment:
azure-storage should only be available in test ?

Reply:
It is :) Line 75 :)